pkg/inputs/snmp/snmp.go

```diff
@@ -160,6 +160,11 @@ func runSnmpPolling(ctx context.Context, snmpFile string, jchfChan chan []*kt.JC
 				if *dumpMibTable {
 					profile.DumpOids(cl)
 				}
+
+				if profile.NoUseBulkWalkAll {
+					device.NoUseBulkWalkAll = true
+					log.Infof("Turning off BulkWalkAll for device via profile.")
```

Comment:
does this log entry also include the `device_name` field?

Reply:
Good call. Updated to add this.